openassessment/xblock/test/test_staff_area.py

```diff
@@ -781,3 +781,50 @@ def _create_submission(item, values, types):
         peer_api.on_start(submission["uuid"])
         workflow_api.create_workflow(submission["uuid"], types)
         return submission
+
+    @scenario('data/self_only_scenario.xml', user_id='Bob')
+    def test_staff_area_student_info_image_submission_names(self, xblock):
```

Comment:
`image`? Not sure why this word is in test name.

Reply:
Fixed test name.